dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeLogsConnector.java

```diff
@@ -69,7 +72,66 @@ public SnowflakeLogsConnector() {
   public enum SnowflakeLogConnectorProperties implements ConnectorProperty {
     OVERRIDE_QUERY("snowflake.logs.query", "Custom query for log dump."),
     OVERRIDE_WHERE(
-        "snowflake.logs.where", "Custom where condition to append to query for log dump.");
+        "snowflake.logs.where", "Custom where condition to append to query for log dump."),
+
```

Comment:
I feel there is not much point in having individual OVERRIDE_WHERE options for each of these tables.
OVERRIDE_QUERY for each table might make sense, but probably also not a necessity right now.

I suggest several options for the OVERRIDE_WHERE:
1. Use the same OVERRIDE_WHERE as query logs.
2. Have "themed" OVERRIDE_WHERE - e.g. filtering by timestamp, filtering by warehouse/database, etc. Note that it is probably not a P0/P1 priority either.